src/fake/token_store.js

```diff
@@ -61,14 +66,39 @@ const createTokenStore = () => {
 
     const token = {
       token: {
-        access_token: generatePasswordAccessToken(username, password),
-        refresh_token: generatePasswordRefreshToken(username, password),
+        access_token: generateAccessToken(username),
+        refresh_token: generateRefreshToken(username),
+        token_type: 'bearer',
+        expires_in: 86400,
+        scope: 'user',
+      },
+      user: {
+        username,
+      },
+    };
+    tokens.push(token);
+
+    return token.token;
+  };
+
+  const createTokenWithAuthorizationCode = authorizationCode => {
+    const knownCode = _.find(knownAuthorizationCodes, ({ code }) => code === authorizationCode);
+
+    if (!knownCode) {
+      return null;
+    }
+
+    const { username } = knownCode;
+    const token = {
+      token: {
+        access_token: generateAccessToken(username),
+        refresh_token: generateRefreshToken(username),
         token_type: 'bearer',
         expires_in: 86400,
+        scope: 'user:limited',
```

Comment:
@lyyder @sktoiva @bladealslayer 
Btw. have you considered other scope names? It would be nice to keep the door open for other kinds of "limited" user-scopes too. It's hard to predict if we need to create more scopes at some point.

Reply:
The discussion regarding the scope naming/hierarchy will still continue.